docs/mobile-apps/features/mobile-app-diagnostics/app-crash-logs.md

```diff
@@ -39,15 +66,18 @@ You can download the crash log as a text file to save and share with Sauce Labs
 
```

Comment:
We should add something about the fact that the link to the crash/error report available in Backtrace can also be sent directly instead of attaching a file.